src/lib/ui/next/customNextPage.js

```diff
@@ -60,6 +118,22 @@ export default function Home() {
             Get started by editing
             <code className={styles.code}> app/page.tsx</code>
           </p>
+          <div className={styles.state}>
+            {error ? (
+              <span className={styles.error}>Error: {error}</span>
+            ) : (
+              <div>
+                <div>Contract State: <span className={styles.bold}>{contractState}</span></div>
```

Comment:
We can see the contract state via minascan with or without a wallet.  Could we put this out of the error ternary, add a connect wallet button, and then only do this error ternary if someone requested the wallet to connect and it failed afterwards?

Reply:
Nvm!